dev-libs/capstone/capstone-6.0.0_alpha2.ebuild

```diff
@@ -74,7 +74,8 @@ src_prepare() {
 
 src_configure() {
 	local mycmakeargs=(
-		-DBUILD_SHARED_LIBS=true
+		-DCAPSTONE_BUILD_SHARED_LIBS=true
+		-DCAPSTONE_BUILD_STATIC_LIBS=false
 		-DCAPSTONE_BUILD_STATIC_RUNTIME=false
```

Comment:
```
 * QA Notice: One or more CMake variables were not used by the project:
 *   CAPSTONE_BUILD_STATIC_RUNTIME
```
It seems they removed both old variables at the same time.

Reply:
fixed